crates/rpc-jsonrpsee/src/traits.rs

```diff
@@ -0,0 +1,162 @@
+#![allow(missing_docs)]
+
+//! Rollup Node
+
+use alloc::{boxed::Box, string::String, vec::Vec};
+use core::net::IpAddr;
+
+use alloy_eips::BlockNumberOrTag;
+use alloy_primitives::{B256, U64};
+use jsonrpsee::{core::RpcResult, proc_macros::rpc};
+use maili_common::{ProtocolVersion, SuperchainSignal};
+use maili_protocol::{ExecutingMessage, SafetyLevel};
```

Comment:
I think we should consider moving those traits to maili as well, because I'd prefer if the repo dependency is one directional

Reply:
yes we are working on removing the dep on op-alloy in maili